articles/ml_system_design/real_time_news_search_using_upstash_kafka_and_vector/upstash_ingest/README.md

```diff
@@ -0,0 +1,55 @@
+Key Points:
+
+## NewsAPI
+For free, developer plan the content is cut after 260 characters.
+With a paid plan, you get the whole article.
```

Comment:
The `README` needs more work.

How to install it?

What extra services are we using?

How to set the dependencies & get the API keys:
- news API
- upstash

What about costs?

How to use it?

Reply:
How about the hardware you need to run this? At least broadly speaking, e.g., you need a GPU or not

Reply:
Readme is a mess - spent 0 time working on it, left it for the end.